### ISSUE TITLE
[expr.dynamic.cast] The phrase "runtime check" is not quite correct

### ISSUE THREAD

**Issue by frederick-vs-ja** [cplusplus/draft#6799]
`dynamic_cast`, which possibly performs the so-called "runtime check", is allowed in constant evaluation since C++20 via [P1327R1](https://wg21.link/p1327r1). Perhaps we should choose another phrase to describe such checks.

**Comment by Eisenwave** [comment]
I fully agree. C++ doesn't really have a concept of runtime or compile-time anyway, so this wording feels very out-of-place in general.

Perhaps something like
```diff
-Otherwise, a runtime check is applied to see if the object pointed or referred to by v can be converted to the type pointed or referred to by T.
+Otherwise, the effect of the expression dynamic_cast<T>(v) is determined as follows.
```
```diff
[...]
-Otherwise, the runtime check fails.
+Otherwise, the cast fails.
```

**Comment by frederick-vs-ja** [comment]
[P3953](https://wg21.link/p3953) changes `runtime_format` to `dynamic_format`, so I think it would be fine and somehow consistent to use "dynamic check" in [expr.dynamic.cast] now.
